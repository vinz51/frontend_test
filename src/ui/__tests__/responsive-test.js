import renderer from "react-test-renderer";
import React from "react";
import Responsive from "../responsive";

describe("Responsive", () => {
  describe("Snapshots", () => {
    it("renders", () => {
      const myTreeComponent = renderer
        .create(
          <Responsive xs={true} sm={true} md={true} lg={true}>
            Show the content
          </Responsive>
        )
        .toJSON();

      expect(myTreeComponent).toMatchSnapshot();
    });
  });
});
