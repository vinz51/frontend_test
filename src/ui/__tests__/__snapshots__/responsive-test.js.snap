// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Responsive Snapshots renders 1`] = `
<div
  className="sc-AxjAm fIpvdq"
>
  Show the content
</div>
`;
